packages/css/src/components/context/context.stories.ts

```diff
@@ -1,13 +1,24 @@
 import { storiesOfContext } from '@dso-toolkit/sources';
 import { storiesOf } from '@storybook/web-components';
 
-import { buttonTemplate } from '../button/button.template';
 import { contextTemplate } from './context.template';
 
 import { html } from 'lit-html';
 import readme from './readme.md';
 
-const content = buttonTemplate({ url: '#', modifier: 'dso-tertiary', label: 'Versies', icon: { icon: 'chevron-down' } });
+const content = html`
+  <div class="form-group dso-select">
```

Comment:
Ik ben toch niet helemaal blij met deze opzet. Een `form-group` is alleen om structuur in een Formulier te brengen.

Net zoals dat `.dso-label-container` en `.dso-field-container` kolommen zijn in een Formulier.

cc: @daverhelst, @gofix en @bramhoosemans

Reply:
Ik snap het bezwaar en inderdaad, een sorteeroptie is niet bepaald een 'formulier', maar ik vind het toch ook wel heel lastig om een 'tegenbod' te doen. Ik heb even een vingeroefeningetje gedaan met een soort van 'subcomponent' dat dan in zo'n `.dso-context-container` zou kunnen worden geplaatst. 

https://github.com/dso-toolkit/dso-toolkit/commit/6b389da72c62544f67d2030d4db452430192d43a

Even afgezien van dat `.dso-context-filter` niet de goede naam zou zijn (het is een sorteerding, dus `.dso-context-sort` zou beter zijn), vind ik het toch lastig hier helemaal achter te staan. Allerlei regeltjes die van toepassing zijn op `.form-group .dso-select` zouden we dan in dat component moeten gaan herhalen (denk aan uitlijning van labeltekstje en de pulldown-optie-tekstjes, waar bootstrap altijd zo z'n best voor doet), of gericht gaan kopieren (al dan niet met mixins).

Een andere optie zou zijn dan maar 'gewoon' label en select toe te staan en de beoogde styling daarop toe te passen, altijd in scope van de `.dso-context-container`. Voelt ook niet goed wat mij betreft.

Overigens vind ik de hele context-component al best een lastig ding hoor, want het enige wat het eigenlijk doet is dingen links en rechts van elkaar zetten (en onder omstandigheden dan weer onder elkaar). Klinkt eenvoudig en generiek. Maar: wat er IN die `.dso-context-container` en `.dso-context-label` staat kan ook nog 'van alles' zijn, en die wil je vaak verticaal zich op een bepaalde manier tot elkaar laten verhouden (om ze te laten lijnen). Maar: dan zou je eigenlijk ook dus styling voor dat 'van alles' moeten uitwerken. Of zie ik dat te problematisch en staan we toch eigenlijk alleen maar toe wat we etaleren...

Wat had jij zelf in gedachten @tfrijsewijk

Reply:
Dit ziet er goed uit! Zoals besproken maken we er een Context-specifieke werkvorm van: `.dso-context-select`.